Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "TimeSpans"
 uuid = "bb34ddd2-327f-4c4a-bfb0-c98fc494ece1"
 authors = ["Beacon Biosignals, Inc."]
-version = "0.3.3"
+version = "0.3.4"
```

Comment:
keep in mind that this bump is competing with #51  - so we should rebump whichever one gets merged second

Reply:
```suggestion
version = "0.3.5"
```